internal/handlers/sqlite/msg_aggregate.go

```diff
@@ -408,9 +408,9 @@ func processStagesStats(ctx context.Context, closer *iterator.MultiCloser, p *st
 				"count", collStats.CountObjects,
 				"avgObjSize", avgObjSize,
 				"storageSize", collStats.SizeCollection,
-				"freeStorageSize", int64(0), // TODO https://github.com/FerretDB/FerretDB/issues/2342
-				"capped", false, // TODO https://github.com/FerretDB/FerretDB/issues/2342
-				"wiredTiger", must.NotFail(types.NewDocument()), // TODO https://github.com/FerretDB/FerretDB/issues/2342
+				// TODO https://github.com/FerretDB/FerretDB/issues/2447
+				"freeStorageSize", int64(0),
+				"capped", collStats.CappedSize != int64(0),
```

Comment:
```suggestion
				"capped", collStats.CappedSize > 0,
```

That's what we use already

Reply:
At the end, this was replaced by .Capped() of CollectionInfo